Protocol.md

```diff
@@ -627,6 +631,13 @@ subprotocol strings, and:
   SHALL validate that the negotiated subprotocol is present in the
   *subprotocols* field.
 
+Furthermore, the server SHALL validate that the *ping_interval* field
+contains a non-negative integer. If the value is `0`, the server SHALL
+NOT send WebSocket *ping* messages to the client. Otherwise, the server
+SHOULD send a WebSocket *ping* message in the requested interval to the
```

Comment:
If we don't rename the field to `ping_interval_seconds`, this line should say `in the requested interval in seconds`.